index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Apple old website clone.</title>
+  <link rel="stylesheet" href="css/styles.css">
+  <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
+</head>
+<body>
+  <header>
+    <nav>
+      <span class="nav-items">
+        <a href="#" class="nav-links">
+          <span class="iconify apple-logo" data-icon="ant-design:apple-filled" data-inline="false"></span>
+        </a>
+      </span>
+
+      <span class="nav-items">
+        <a href="#" class="nav-links">
+          <span>Store</span>
```

Comment:
This ```<span>``` wrapper is unnecessary. This applies to the other elements that match this pattern in the navbar. Except for the apple, and search icon that uses iconify syntax.